### PR TITLE
DOCS-431: Add reference article about GELI

### PR DESCRIPTION
**Pull request by Mrt134**
Thanks for contributing to FreeNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/en/docs/ui-reference-guide/GELI.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+cache) and SLOG (write cache). Drives added to an existing encrypted
+pool are encrypted with the same method specified when the pool was
+created. Data in memory, including ARC, is not encrypted. ZFS data on
+disk, including L2ARC and SLOG, are encrypted if the underlying disks
+are encrypted. Swap data on disk is always encrypted.
+
+Encryption performance depends upon the number of disks encrypted. The
+more drives in an encrypted pool, the more encryption and decryption
+overhead, and the greater the impact on performance. **Encrypted pools
+composed of more than eight drives can suffer severe performance
```
Comment (q5sys) [inline content/en/docs/ui-reference-guide/GELI.md:47]: Verify this, I believe this was only true of the older GELI encryption method.  I don't know if the newer one has the same issue.

**Comment by Mrt134** [comment]
Ok, the goal is to have this article be solely about the GELI method, so I'll do another pass on it and add that terminology wherever possible.

**Comment by q5sys** [comment]
> Ok, the goal is to have this article be solely about the GELI method, so I'll do another pass on it and add that terminology wherever possible.

this may be an issue of reading this on github without formatting, so any statement that specified that this was only about GELI didn't stand out as much.  If those two intro lines have a stronger formatting style than the rest of the document it might not be as confusing to others as I worry it might be.

File: content/en/docs/ui-reference-guide/GELI.md (hunk 1 above)
Comment (yocalebo) [reply content/en/docs/ui-reference-guide/GELI.md:47]: This is all true related to GELI except for the last sentence that says `encrypted pools composed of more than eight drives can suffer severe performance penalties` We have multiple truenas customers that are using 50+ disk in GELI encrypted zpools with no perceived performance issues. I would suggest that we leave any specific HDD numbers off the documentation for 2 reasons.
1. our customers will read this and think they have a problem and say "well, I've got 75 disks in my pool" so I must have a problem.
2. the actual number of HDD's before GELI encryption penalties are seen is, quite literally, never been quantified in any scientific manner so it's all just hear-say.

As a matter of fact, the performance team did tests with a 100 disk GELI pool and the performance numbers were almost as fast as a pool with no GELI encryption. I'm being long-winded to say, I'd recommend you remove the 8 drive number as that will only cause confusion.

Comment (Mrt134) [reply content/en/docs/ui-reference-guide/GELI.md:47]: Can do!

**Pull request merged**